sdk-tests/src/test/java/io/dapr/it/testcontainers/actors/DaprActorsIT.java

```diff
@@ -20,64 +20,32 @@
 import io.dapr.testcontainers.Component;
 import io.dapr.testcontainers.DaprContainer;
 import io.dapr.testcontainers.DaprLogLevel;
+import io.dapr.testcontainers.spring.DaprContainerFactory;
+import io.dapr.testcontainers.spring.DaprSidecarContainer;
+import io.dapr.testcontainers.spring.DaprSpringBootTest;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Tag;
 import org.junit.jupiter.api.Test;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.test.context.SpringBootTest;
-import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
-import org.springframework.test.context.DynamicPropertyRegistry;
-import org.springframework.test.context.DynamicPropertySource;
-import org.testcontainers.containers.Network;
 import org.testcontainers.containers.wait.strategy.Wait;
-import org.testcontainers.junit.jupiter.Container;
-import org.testcontainers.junit.jupiter.Testcontainers;
 
 import java.util.Map;
-import java.util.Random;
 import java.util.UUID;
 
-import static io.dapr.it.testcontainers.ContainerConstants.DAPR_RUNTIME_IMAGE_TAG;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
-@SpringBootTest(
-        webEnvironment = WebEnvironment.RANDOM_PORT,
-        classes = {
-                TestActorsApplication.class,
-                TestDaprActorsConfiguration.class
-        }
-)
-@Testcontainers
+@DaprSpringBootTest(classes = {TestActorsApplication.class, TestDaprActorsConfiguration.class})
```

Comment:
@artur-ciocanu I am surprised that this is the only tests that can be changed.. did you added this here to show it it will work?